### PR TITLE
feat: introduce car explorer (CAR import)

### PR DESCRIPTION
**Pull request by alvin-reyes**
# Changes

This change introduce the `<IpldCarExplorerForm />` on the explorer.ipld.io. 

# Related PR
https://github.com/ipfs/ipld-explorer-components/pull/313

# Verification
image
image
image

### CODE REVIEW

File: src/components/header/header.css [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+.cidlookup:hover{
+  background-color: #fff;
+  color: #69c4cd;
+}
+
+.cidlookup:hover:after {
+  opacity: 1;
+  right: 10px;
+}
```
Comment (lidel) [inline src/components/header/header.css:36]: nit: please remove this custom css file and style using CSS classes from Tachyons (like we do everywhere):
- https://tachyons.io/docs/
- https://github.com/ipfs-shipyard/ipfs-css#readme

**Review by lidel — CHANGES_REQUESTED** [review]
LGTM, switched to ipld-explorer-components@2.1.0 and cleaned up some locale deps.


One thing to fix before we merge is UX  of selecting a CAR:

-  when CAR file is selected, nothing happens. Took me a moment to realize I need to click "Explore"
  - this is confusing, no need for this additional click – we should open the root CID as soon `dag.import` is finished.

**Comment by alvin-reyes** [comment]
The issue here will be resolved by PR https://github.com/ipfs/ipld-explorer-components/pull/317

File: src/components/header/Header.js [hunk 2]
```diff
@@ -1,17 +1,27 @@
...
+
   return (
     <header className='flex-l items-center pa3 bg-navy bb bw3 border-aqua tc tl-l'>
       <a href='#/' title={t('homeLink')} className='flex-none v-mid'>
         <img src={ipfsLogo} alt='IPFS' style={{height: 50, width: 117.5}} />
       </a>
-      <div className='flex-auto ph2 ph3-l pt1'>
+      <div className="btn-group" style={{ marginLeft: '15px'}}>
+        <button onClick={handleOnChange} value="cid" className="f6 link dim br2 ba ph3 pv2 mb2 dib navy" style={{ border: '0px', backgroundColor: '#69c4cd', borderRight: '1px solid white', marginTop: '5px', borderRadius: '5px 0px 0px 5px' }} aria-current="page">CID</button>
+        <button onClick={handleOnChange} value="car" className="f6 link dim br2 ba ph3 pv2 mb2 dib navy" style={{ border: '0px', backgroundColor: '#69c4cd', marginTop: '5px', borderRadius: '0px 5px 5px 0px'}} aria-current="page">CAR</button>
```
Comment (lidel) [inline src/components/header/Header.js:20]:
```suggestion
      <div className="btn-group" className='ph2 ph3-l'>
        <button onClick={handleOnChange} value="cid" className="f6 pointer dim br2 ba ph3 pv2 mb2 dib navy" style={{ border: '0px', backgroundColor: '#69c4cd', borderRight: '1px solid #0b3a53', marginTop: '5px', borderRadius: '5px 0px 0px 5px' }} aria-current="page">CID</button>
        <button onClick={handleOnChange} value="car" className="f6 pointer dim br2 ba ph3 pv2 mb2 dib navy" style={{ border: '0px', backgroundColor: '#69c4cd', marginTop: '5px', borderRadius: '0px 5px 5px 0px'}} aria-current="page">CAR</button>
```

**Review by lidel — APPROVED** [review]
Works as expected now :+1:  

Let's merge, we can improve UI in separate PR.

**Pull request merged**